ppr/doc/libpaf-ppr.3

```diff
@@ -0,0 +1,122 @@
+.\" Copyright IBM Corp. 2015
+.\"
+.\" The MIT License (MIT)
+.\"
+.\" Permission is hereby granted, free of charge, to any person obtaining a copy
+.\" of this software and associated documentation files (the "Software"), to
+.\" deal in the Software without restriction, including without limitation the
+.\" rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+.\" sell copies of the Software, and to permit persons to whom the Software is
+.\" furnished to do so, subject to the following conditions:
+.\"
+.\" The above copyright notice and this permission notice shall be included in
+.\" all copies or substantial portions of the Software.
+.\"
+.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+.\" AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.\" LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+.\" FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+.\" IN THE SOFTWARE.
+.\"
+.\" Contributors:
+.\"     IBM Corporation, Adhemerval Zanella - Initial implementation.
+.\"     IBM Corporation, Rajalakshmi S - Initial implementation.
+.TH LIBPAF 2015-08-06 "Linux" "Program Priority Register"
+.SH NAME
+The Power Architecture Facilities - Program Priority Register
+.SH SYNOPSIS
+.sp
+.BI "#include <paf/ppr.h>
```

Comment:
I just noticed a few things, here.

1) This whole line is already in bold, and it should indeed be. Then, there is no need to use .BI.
2) If you use .B, then you could get rid of the quote.

I think this line should be:

.B #include <paf/ppr.h>